src/AppBundle/Membership/AdherentChangePasswordHandler.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace AppBundle\Membership;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Security\Core\Encoder\EncoderFactoryInterface;
+
+class AdherentChangePasswordHandler
+{
+    private $manager;
+    private $encoderFactory;
+
+    public function __construct(ObjectManager $manager, EncoderFactoryInterface $encoderFactory)
```

Comment:
Well I agree with Romain as we already have the user so we can use the `security.password_encoder` service and inject the `UserPasswordEncoderInterface` service. Probably easier to mock in unit tests.